CHANGELOG.md

```diff
@@ -690,6 +695,10 @@ Version 2.0 of the UI framework is built for player 7.1. If absolutely necessary
 ## 1.0.0 (2017-02-03)
 - First release
 
+[3.21.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.20.0...v3.11.0
```

Comment:
Shouldn't that be for `v3.21.0` instead?

```suggestion
[3.21.0]: https://github.com/bitmovin/bitmovin-player-ui/compare/v3.20.0...v3.21.0
```

Reply:
Thanks for catching this. Updated.